core/src/components/input/input.ionic.scss

```diff
@@ -24,10 +25,18 @@
 // ----------------------------------------------------------------
 
 .input-bottom .helper-text {
-  // TODO(FW-6112): Verify the design token is correct once it's available and remove the hardcoded value.
-  color: var(--ionic-color-neutral-600, #535353);
+  color: #{tokens.$ionic-color-neutral-600};
```

Comment:
@thetaPC I updated the TODO for FW-6112 here to use the design token, is that fine? I checked and it is the same value:

```
$ionic-color-neutral-600: var(--ionic-color-neutral-600, #535353);
```

Reply:
That works for me. I also have it updated on my success state PR. Let's see who gets merged first? 😆